src/coord/timestamp.rs

```diff
@@ -62,13 +134,37 @@ enum RtTimestampConnector {
 /// Timestamp consumer: wrapper around source consumers that stores necessary information
 /// about topics and offset for byo consistency
 struct ByoTimestampConsumer {
+    // Source Connector
     connector: ByoTimestampConnector,
+    // The name of the source with which this connector is associated
     source_name: String,
+    // The format of the connector
+    envelope: ConsistencyFormatting,
+    // The last timestamp assigned per partition
     last_partition_ts: HashMap<i32, u64>,
+    // The max assigned timestamp. Should be max(last_partition_ts)
     last_ts: u64,
+    // The max offset for which a timestamp has been assigned
+    last_offset: i64,
+    // The total number of partitions for the data topic
     current_partition_count: i32,
 }
 
+/// Supported format/envelope pairs for consistency topic decoding
+/// TODO(natacha): this should be removed
+enum ConsistencyFormatting {
+    // The formatting of this consistency source is currently unknown
+    Unknown,
+    // The formatting of this consistency source follows the
+    // SourceName,PartitionCount,PartitionId,TS,Offset
+    Raw,
+    // The formatting of this consistency source follows the
+    // Debezium Kafka format
+    DebeziumKafka,
+    // The formatting of this consistency source follows the
+    // Debezium format (OCF + Avro)
+    DebeziumOcf,
```

Comment:
I'm confused about this variant. Is Debezium really putting OCF files into Kafka? Or is it just omitting the schema registry header bit, and splatting down the raw Avro bytes with the OCF container?

Reply:
I think the TB version (based on context).

Reply:
Oh I see, that won’t go into Kafka at all. Makes sense.

Reply:
The consistency topic generated by Debezium Embedded Connector (the one that we use in TB) generates OCF files.